src/ManageCourses.Api/Model/UcasCourseEnrichmentGetModel.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using GovUk.Education.ManageCourses.Domain.Models;
+
+namespace GovUk.Education.ManageCourses.Api.Model
+{
+    public class UcasCourseEnrichmentGetModel
+    {
+        public UcasCourseEnrichmentGetModel()
+        {
+            EnrichmentModel = new CourseEnrichmentModel();
+        }
+        public CourseEnrichmentModel EnrichmentModel { get; set; }
```

Comment:
Doesn't this need identifying informmation aka. a `courseCode` and a `providerCode`?

Reply:
They are in the url, captured as separate string parameters.
Just tried out some variations locally with the fields added to the model.

* Adding to model: now have duplicate places to post this info, bits of url are not bound to fields of model
* Adding to model and removing controller action params: results in broken url pattern (replacement bits disappear, losing our nice restful url setup)